portfolio/src/main/webapp/style.css

```diff
@@ -124,10 +125,16 @@ button:active{
 #eas{
     background-image:url(/images/easSweatshirt.png);
 }
+#eas2012{
+    background-image:url(/images/eas2012.png);
+}
 #traditional{
-    background-image:url(Passionfruit.png);
+    background-image:url();
     /*replace once photograph art*/
 }
```

Comment:
if this is not being used, we should get rid of it.

Reply:
I just uploaded the new images, so it will be used in the next fix.